.github/workflows/dashboards-test.yml

```diff
@@ -27,7 +27,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: [lts]
+        node-version: [18.x]
```

Comment:
Why the change?

Reply:
In the initial commit it gave the following error: 

```sh
Run actions/setup-node@v3
Attempting to download lts...
Not found in manifest. Falling back to download directly from Node
Error: Unable to find Node version 'lts' for platform linux and architecture x64.
```

Which I took to mean that the `node-version` does not support just `lts` (but according to the docs it _should_ support `lts/{codename}`. I see now that this workflow does not use the setup-node action, so that would explain why it did not fail previously.